test/unit/three_scale/json_validator_test.rb

```diff
@@ -16,17 +16,15 @@ class ThreeScale::JSONValidatorTest < ActiveSupport::TestCase
 
   test 'extract the schema draft' do
     assert_equal 4, @klass.send(:schema_draft, { '$schema' => 'http://json-schema.org/draft-04/schema#' })
-    assert_equal 7, @klass.send(:schema_draft, { '$schema' => 'http://json-schema.org/draft-07/schema#' })
     assert_equal 0, @klass.send(:schema_draft, {})
   end
 
   test 'extract the schema id' do
     assert_equal 'my-id', @klass.send(:schema_id, { 'id' => 'my-id', '$schema' => 'http://json-schema.org/draft-04/schema#' })
-    assert_equal 'my-id', @klass.send(:schema_id, { '$id' => 'my-id', '$schema' => 'http://json-schema.org/draft-07/schema#' })
   end
 
   test 'schema build' do
-    schema_json = { '$id' => 'http://example.com/schema-v1/my-schema#frag', '$schema' => 'http://json-schema.org/draft-07/schema#' }
+    schema_json = { 'id' => 'http://example.com/schema-v1/my-schema#frag', '$schema' => 'http://json-schema.org/draft-04/schema#' }
```

Comment:
I'm a bit confused about JSON versions and everything. We only support up to v4 now? is that not a problem? 😵‍💫

Reply:
up to v4 support is for `ThreeScale::JSONValidator` which depends on `json-schema` gem.

it is OK because we're going to use it only for validating the older swagger specs.